lib/senders/email.js

```diff
@@ -337,10 +337,11 @@ module.exports = function (log, config) {
           mailer.sendMail(emailConfig, (err, status) => {
             if (err) {
               log.error({
-                op: 'mailer.send.1',
-                err: err && err.message,
-                status: status && status.message,
-                id: status && status.messageId,
+                op: 'mailer.send.error',
+                err: err.message,
+                code: err.code,
+                errno: err.errno,
+                message: status && status.message,
```

Comment:
The `message` field that comes back in the response can contain useful information for debugging, hence including both it and `err.message`.

Reply:
I wonder if this was in place because PII might be logged in the message. Is there any risk of that?

Reply:
Oh, no, misleading property name, sorry. `message` in this case is the raw error message from SES, for instance:

```
"message":"Unknown(\"<ErrorResponse xmlns=\\\"http://ses.amazonaws.com/doc/2010-12-01/\\\">\\n  <Error>\\n    <Type>Sender</Type>\\n    <Code>MessageRejected</Code>\\n    <Message>Email address is not verified. The following identities failed the check in region US-EAST-1: Firefox Accounts &lt;pmbooth@gmail.com&gt;</Message>\\n  </Error>\\n  <RequestId>83bacdcf-9a32-11e8-b030-0dcd9a8a0b2f</RequestId>\\n</ErrorResponse>\\n\")"
```